cpp/order/Steinhardt.h

```diff
@@ -157,6 +157,13 @@ class Steinhardt
     void aggregateWl(std::shared_ptr<std::complex<float>> target,
                      std::shared_ptr<std::complex<float>> source);
 
+    //! Compute array index for m values, indexed like:
+    //  [0, 1, ..., l, -1, -2, ..., -l]
+    int mIndex(int m) const
+    {
+        return m < 0 ? m_l - m : m;
+    }
+
```

Comment:
@bdice @klarh Can we make a PR to fsph that creates a C++ function equivalent to the Python function `get_LMs` that it exposes? It would make a freud's indexing w.r.t. spherical harmonics much more transparent if fsph actually told you the order in which it returned spherical harmonics, ideally via a simple iterator.

Reply:
That's a great idea -- I considered doing that but I felt it was out of scope for this PR. I created https://github.com/glotzerlab/fsph/issues/1.

Reply:
Agreed, that can be a separate PR.